src/Jobs/PtpJob.php

```diff
@@ -0,0 +1,236 @@
+     *
+     * @var integer
+     */
+    public $tries = 1;
+
+    /**
+     * The job ID.
+     *
+     * @var string
+     */
+    public $id;
+
+    /**
+     * The user who submitted the Largo session.
+     *
+     * @var \Biigle\User
+     */
+    public $user;
+
+    /**
+     * Array of all dismissed image annotation IDs for each label.
+     *
+     * @var array
+     */
+    public $dismissedImageAnnotations;
+
+    /**
+     * Array of all changed image annotation IDs for each label.
+     *
+     * @var array
+     */
+    public $changedImageAnnotations;
+
+    /**
+     * Array of all dismissed video annotation IDs for each label.
+     *
+     * @var array
+     */
+    public $dismissedVideoAnnotations;
+
+    /**
+     * Array of all changed video annotation IDs for each label.
+     *
+     * @var array
+     */
+    public $changedVideoAnnotations;
+
+    /**
+     * Whether to dismiss labels even if they were created by other users.
+     *
+     * @var bool
+     */
+    public $force;
+
+    //TODO: Fix comments below and remove the useless ones above
+
+    /**
+     * ID of the image where PTP is applied
+     *
+     * @var int
+     */
+    public int $imageId ;
+
+    /**
+     * Type of the Job between compute-area and ptp
+     *
+     * @var string
+     */
+    public string $jobType;
+
+
+    /**
+     * Array of annotations
+     *
+     * @var array
+     */
+    public array $annotations ;
+
+    /**
+     * Array of annotation coordinates (Points)
+     *
+     * @var array
+     */
+    public array $points ;
+
+    /**
+     * ID of the label of the annotation to convert
+     *
+     * @var int
+     */
+    public int $labelId ;
+    /**
+     * Output file where the PTP job output will be stored
+     *
+     * @var string
+     */
+    public string $outputFile;
+
+    /**
+     * IDs of the annotations to convert
+     *
+     * @var array
+     */
+    public array $annotationIds;
+
+    /**
+     * Create a new job instance.
+     *
+     * @param \Biigle\User $user
+     * @param ImageAnnotation[] $annotations
+     *
+     * @return void
+     */
+    public function __construct(User $user, array $annotations, string $jobType, int $labelId, string $outputFile)
+    {
+        $this->queue = config('ptp.job_queue');
+        $this->targetDisk = config('ptp.ptp_storage_disk');
+        $this->user = $user;
+
+        //Assumes that annotations are grouped by images
+        $this->imageId = $annotations[0]['image'];
+        $this->outputFile = $outputFile;
+
+        // We expect these annotations to be point annotations
+        $this->points = [];
+        $this->labelId= $labelId;
+        $this->annotationIds = [];
+        $this->jobType = $jobType;
+        foreach ($annotations as $annotation) {
+            $this->points[] = implode(',', $annotation['points']);
+            //TODO: find a better way to pass this arg.
+            $this->annotationIds[] = $annotation['annotation_id'];
```

Comment:
In the past when I had too many input parameters to a Python script I would write everything into a JSON file and pass this to the Python script.

With the annotation IDs as inline parameter you could easily run into the CLI parameter limit.